geograypher/utils/numeric.py

```diff
@@ -94,7 +182,7 @@ def triangulate_rays_lstsq(starts, directions):
     A = np.concatenate(As, axis=0)
     b = np.concatenate(bs, axis=0)
 
-    x, _, _, _ = np.linalg.lstsq(A, b)
+    x, _, _, _ = np.linalg.lstsq(A, b, rcond=None)
```

Comment:
Out of curiosity, what issues were you having with the default rcond? `None` looks like the default in at least one version of `scipy` ([here](https://numpy.org/doc/2.1/reference/generated/numpy.linalg.lstsq.html))